### DISCUSSION TITLE
Inquiry Regarding Open Source Project and Contribution Opportunities

### DISCUSSION THREAD

**Discussion by Raza9798** [lakasir/lakasir#143] [Q&A] [answered]
As an advocate for open-source collaboration, I'm reaching out to inquire about the project's status and whether it's open for contributions.

If Lakasir is indeed an open-source project, I'd love to learn more about its implementation plan and the features that are pending implementation. I'm keen on actively contributing to its development and would appreciate any guidance or information you can provide regarding the project's roadmap and areas where additional help is needed.

Looking forward to your response and potentially collaborating on this exciting project.

**Comment by sheenazien8** [comment, marked as answer]
Hey @Raza9798 ya it is an  open source project, right now lakasir already has these features

- register user -> tenant
- product management
- member management
- category management
- profile
- transaction (WIP)


The pending features for the app are:
- dashboard
- advance report
- unit price = the product has and each unit prices
- User management 
- Role management


And my ideas for the web apps are:
- bulk insert for product
- role management
- disable enabled client

Thanks for your appreciation, I'm pleased if you want to collaborate on this project, you can install the APK first by building the project from https://github.com/lakasir/lakasir_flutter, and if you're going to download the APK directly you can download it in [here](https://drive.usercontent.google.com/download?id=1U5cXfzDOARKhWu6TRzNDQ6zMqFdsPJVb&export=download&authuser=0)


then you can join on this [telegram group](https://t.me/+aZqCuQSUEmQxMTc1)

**Reply by Raza9798** [reply]
Greate i will contribute on the backend with Laravel and API @sheenazien8

**Reply by sheenazien8** [reply]
Appreciate it man, You can keep an eye on this [project](https://github.com/orgs/lakasir/projects/2/views/1) @Raza9798

**Reply by Raza9798** [reply]
okay can i contact you on whatsapp?

**Reply by sheenazien8** [reply]
sent, got your number from your profile @Raza9798
